_search-plugins/star-tree-index.md

```diff
@@ -73,7 +73,7 @@ To use a star-tree index, modify the following settings:
 
 ## Example mapping
 
-In the following example, index mappings define the star-tree configuration. This star-tree index precomputes aggregations in the `log` index. The aggregations are calculated using the `size` and `latency` fields for all the combinations of values indexed in the `port` and `status` fields:
+In the following example, index mappings define the star-tree configuration. This star-tree index precomputes aggregations in the `logs` index. The aggregations are calculated using the `size` and `latency` fields for all the combinations of values indexed in the `port` and `status` fields:
```

Comment:
Nit: 
In the following example, index mappings define the star-tree configuration. This star-tree index precomputes aggregations in the `logs` index. The aggregations are calculated for the `size` and `latency` fields for all the combinations of values indexed in the `port` and `status` fields: